tests/cross_fw/examples/example_scope.json

```diff
@@ -207,5 +207,27 @@
             "ratio": 1.0,
             "group_size": 64
         }
+    },
+    "quantization_aware_training_torch_anomalib": {
+        "backend": "torch",
+        "requirements": "examples/quantization_aware_training/torch/anomalib/requirements.txt",
+        "cpu": "Intel(R) Core(TM) i9-10980XE CPU @ 3.00GHz",
+        "accuracy_tolerance": 0.02,
+        "accuracy_metrics": {
+            "fp32_f1score": 0.9919999837875366,
+            "int8_init_f1score": 0.9767441749572754,
+            "int8_f1score": 0.9919999837875366,
+            "accuracy_drop": 0.0
+        },
```

Comment:
Let's unify this test with the latest fix https://github.com/openvinotoolkit/nncf/pull/2594

Reply:
done